src/main/groovy/com/codebrig/arthur/observe/structure/literal/PythonLiteral.groovy

```diff
@@ -17,18 +18,43 @@ class PythonLiteral extends StructureLiteral {
         switch (Objects.requireNonNull(node).internalType) {
             case "Num":
                 return numberValueLiteral()
+            case "Str":
+                return stringValueLiteral()
             default:
                 return null
         }
     }
 
     @Override
     List<String> getPossibleNodeLiteralAttributes(SourceNode node) {
-        def literalAttribute = getNodeLiteralAttribute(node)
-        if (literalAttribute == null) {
-            return Collections.emptyList()
-        } else {
-            return Collections.singletonList(literalAttribute)
+        switch (Objects.requireNonNull(node).internalType) {
+            case "Num":
+                return [numberValueLiteral()]
+            case "Str":
+                return [stringValueLiteral()]
+            default:
+                return Collections.emptyList()
+        }
+    }
+
+    @Override
+    Object getNodeLiteralValue(SourceNode node) {
+        boolean isNegative = node.parentSourceNode.children.any { it.roles.any { it.negative } }
```

Comment:
This should be using `isNodeLiteralNegative()` instead of an inline check.

Reply:
Resolved with commit https://github.com/CodeBrig/Arthur/pull/79/commits/51f0884fa52bfb80b0d83652785a5198d3fff374